main.js

```diff
@@ -1693,6 +1715,8 @@ function execute(script, name, verbose, debug) {
     script._id = Math.floor(Math.random() * 0xFFFFFFFF);
     script.subscribes = {};
     script.subscribesFile = {};
+    script.setStatePerMinuteCounter = 0;
```

Comment:
where is this counter increased? maybe I am just blind

Reply:
sandbox.js -> https://github.com/ioBroker/ioBroker.javascript/pull/1170/files#diff-e3370b4f11191861935af54f280f4be0f3e7e279c042c9a4a8db1cec0cf29393R442